### PR TITLE
Add compilers

### PR DESCRIPTION
**Pull request by dpryan79**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by dpryan79** [comment]
@conda-forge-admin please rerender

**Comment by dpryan79** [comment]
Weird, I guess the bot can't rerender really old forks. I'll do that locally.

**Comment by dpryan79** [comment]
@conda-forge-admin please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -39,15 +45,11 @@ about:
...
   summary: 'The bee swarm plot is a one-dimensional scatter plot like "stripchart", but with closely-packed, non-overlapping points. '
-
   license_family: OTHER
-
   license_file:
     - {{ environ["PREFIX"] }}/lib/R/share/licenses/Artistic-2.0
 extra:
   recipe-maintainers:
     - conda-forge/r
     - jdblischak
```
Comment (cbrueffer) [inline recipe/meta.yaml]: `jdblischak` is part of `conda-forge/r` and can be removed here.

**Pull request merged**